.gitignore

```diff
@@ -8,3 +8,4 @@ build/
 .factorypath
 .vscode
 *.iml
+client_secret.json
```

Comment:
If the app won't work without this file, could you put instructions how to obtain it somewhere?

Reply:
Added to main README under 'Additional Files needed'